egs/ami/s5/cmd.sh

```diff
@@ -12,6 +12,7 @@
 
 export train_cmd="queue.pl --mem 1G"
 export decode_cmd="queue.pl --mem 2G"
+export tensorflow_cmd="queue.pl -l hostname=b*"
```

Comment:
let's call this tfrnnlm_cmd, I don't want people to assume that this is about acoustic models.